### PR TITLE
Created book card

### PR DESCRIPTION
**Pull request by sundstromalbin**
The book card is mostly done, just need to connect it to cremona so the cards display books

### CODE REVIEW

**Review by Kexon — CHANGES_REQUESTED** [review]
bokhajen 🤔

File: app/src/components/Cards.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            sx={{
+              marginLeft: '5%',
+              marginTop: '5%',
+              width: '90%',
+              height: '100%',
+              paddingTop: 5,
+              paddingBottom: 5,
+            }}
+            component="img"
+            image={bok}
```
Comment (Kexon) [inline app/src/components/Cards.tsx:28]: 🤔

**Review by Kexon — APPROVED** [review]
LGTM

File: app/src/components/Cards.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import Typography from '@mui/material/Typography';
+import { CardActionArea, Tooltip } from '@mui/material';
+import bok from '../assets/images/bok.png';
+import { mainTheme } from '../theme';
+
+function Cards() {
+  const text =
+    'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolores libero molestiae consequuntur fuga magni ab. Facilis corporis magni sapiente vel.';
+  return (
+    <div>
```
Comment (emilsvennesson) [inline app/src/components/Cards.tsx:13]: React fragments or Box component

File: app/src/components/Cards.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      <Card>
+        <CardActionArea
+          sx={{ '&:hover': { color: mainTheme.palette.secondary.main } }}
+        >
+          <CardMedia
+            sx={{
+              marginLeft: '5%',
```
Comment (emilsvennesson) [inline app/src/components/Cards.tsx:20]: % is a bit sus

**Pull request merged**